content/games/factions/locations.md

```diff
@@ -0,0 +1,499 @@
+must place the Minotaur within that many inches of the center of the maze.
+
+### Light
+
+Any Group may activate freely to strike or douse a Light. Once they strike a Light, the Group is
+considered to be carrying Light until they douse it.
+
+Groups not carrying Light move at half speed and cannot see more than 6" through the dark. When
+Shooting at a Group that is not carrying Light, reroll hits.
+
+When two Groups fight in melee and neither side is carrying Light, reroll all hits.
+
+Groups carrying Light can see up to 12", and can be seen by any Group which has line of sight, even
+if they are in the dark themselves.
+
+When any Group carrying Light enters a plaza, roll 1d6. For that many turns, all Groups in the plaza
+count as a Group carrying Light while they remain in the plaza.
+
+### The Minotaur
+
+> _The Minotaur guards this maze, hunting anyone carrying Light or just foolish enough to get too_
+> _close_.
+
+The Minotaur acts after each side's activation. It must activate to Attack the nearest Group
+carrying Light if possible, or the nearest Group not carrying Light otherwise. If the Minotaur
+cannot Attack a Group on its turn, it Moves toward the nearest Group carrying Light instead if
+possible, or toward the nearest Group not carrying Light otherwise.
+
+|      ME      |  MI   |  MO   |  FS   |   R   |   T   |                      Traits                       |
+| :----------: | :---: | :---: | :---: | :---: | :---: | :-----------------------------------------------: |
+| 3+ / 3+ / 4+ |   -   |  10"  |   6   |   3   |   5   | Defiant, Lightbane, Reckless, Terrifying, Vicious |
+
+Defiant
+: If this Group is not Shaken and not already in melee, when an enemy successfully activates to
+  Attack them, they may test at 7+ to meet the enemy halfway. Count both Groups as attacking for
+  to-hit rolls.
+
+Lightbane
+: This Group ignores the Terrifying trait for enemies carrying Light, and may reroll to hit them.
+
+Reckless
+: If this Group is not Shaken, it must activate to Attack, if possible.
+
+Terrifying
+: While being Attacked by this Group, enemies must reroll their highest die when testing Resolve.
+
+Vicious
+: When this Group rolls to hit, each 6 inflicts two hits instead.
+
+## 5. Skyboard
+
+> _Airships and gliders fill the air. The Leviathan sings softly, clouds batted aside by its great_
+> _tail._
+
+### Setup
+
+Before placing terrain, both sides roll 2d6. Each side places that many airships (6" long and 3"
+wide) anywhere where they may deploy their Groups. After placing airships, the side with fewer
+airships (or the Defender, if tied) places the Leviathan (12" long and 6" wide) anywhere within 12"
+of the center of the field.
+
+Note that the entire field is impassable terrain as this skirmish occurs in the sky. Groups may
+cross the open air with the Glide activation. For more information, see
+[Sky Fighting](#sky-fighting).
```

Comment:
```suggestion
The entire field is impassable terrain—this skirmish takes place in the sky. Groups may
cross the open air with the Glide activation (see
[Sky Fighting](#sky-fighting)).
```

Reply:
i think ive styled this right, but do check it: i want "(see Sky Fighting)." at the end of the sentence, with the hyperlink and all intact.

Reply:
That's correct!